libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/StatsTableUtils.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ErrorCohort, ILabeledStatistic } from "@responsible-ai/core-ui";
+import { PointOptionsObject } from "highcharts";
+import { IDropdownOption } from "office-ui-fabric-react";
+
+export function generateCohortsStatsTable(
+  cohorts: ErrorCohort[],
+  selectableMetrics: IDropdownOption[],
+  labeledStatistics: ILabeledStatistic[][],
+  selectedMetrics: string[]
+) {
+  const items: PointOptionsObject[] = cohorts.map(
+    (errorCohort, cohortIndex) => {
+      return {
+        colorValue: 0,
+        value: errorCohort.cohortStats.totalCohort,
+

```

Comment:
nit: remove extra line if not needed

Reply:
I wonder where that came from... Thanks!